Chandra/Time/Time.py

```diff
@@ -189,10 +189,6 @@
 import six
 import numpy as np
 
-from . import _axTime3 as axTime3
-
-__version__ = '3.20'
```

Comment:
Hah.  I was trying to see how this commit removed an extra version of _axtime3 ...